TestCases/optimization_euler/equivalentarea_naca64206/NACA64206_adjoint.cfg

```diff
@@ -0,0 +1,159 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA64-206 in inviscid, supersonic flow                  %
+% Author: Yuki Utsumi	      	      	              	       	       %
+% Institution: Individual				               %
+% Date: 2021.07.19                                                             %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES)
+SOLVER= EULER
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+MATH_PROBLEM= DISCRETE_ADJOINT
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART= NO
+
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 1.6
+%
+% Angle of attack (degrees)
+AOA= 0.0
+%
+% Side-slip angle (degrees)
+SIDESLIP_ANGLE= 0.0
+%
+% Free-stream pressure (101325.0 N/m^2 by default, only for Euler equations)
+FREESTREAM_PRESSURE= 11662.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 216.6499
+
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+REF_LENGTH= 1
+REF_AREA= 0
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% ------------------------ SUPERSONIC SIMULATION ------------------------------%
+% MARKER_NEARFIELD needs to be defined on a circumferential boundary within
+% calculation domain so that it captures pressure disturbance from the model.
+% The boundary should have a structured grid with the same number of nodes
+% along each azimuthal angle.
```

Comment:
Perfect! Thanks!

Reply:
@snow54 please add/update the explanation in config_template.cfg in the code root. in case the option is not present please add the option

Reply:
The same explanation as one in configuration file for the test case has been added.